javagen/src/main/java/com/azure/autorest/template/ServiceClientBuilderTemplate.java

```diff
@@ -122,9 +122,29 @@ public final void write(ServiceClient serviceClient, JavaFile javaFile) {
                     javaBlock.line("this.pipelinePolicies = new ArrayList<>();");
                 });
             }
+
+            // ServiceVersion in builder
+            List<ServiceClientProperty> serviceClientProperties = new ArrayList<>(serviceClient.getProperties());
```

Comment:
Should this logic be moved to ServiceClientMapper?

Reply:
The properties in service client are also used in `ClientImpl`. I need `apiVersion` in `ClientImpl` and `serviceVersion` in `ClientBuilder`. I create a new list here. `serviceClient` remains unmodified. We can also add `builderProperties` in `serviceClient`, then template code can be simplified.

Reply:
Is there a reason you need to keep it as `apiVersion` in `ClientImpl`? I know we send it as `api-version` on the wire but locally we can name everything as `serviceVersion` right?

Reply:
I changed `apiVersion` to `serviceVersion` in `ClientImpl` and `ClientBuilder`. Proxy method keeps `apiVersion`.